discovery-provider/src/schemas/track_schema.json

```diff
@@ -0,0 +1,291 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$ref": "#/definitions/Track",
+    "definitions": {
+        "Track": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "blockhash": {
+                    "type": "string"
+                },
+                "blocknumber": {
+                    "type": "integer"
+                },
+                "track_id": {
+                    "type": "integer"
+                },
+                "is_current": {
```

Comment:
I'm not sure how I feel about adding fields that are not part of the metadata on chain (is_current, created_at, and updated_at). So helper fields that are specific to the disc prov should eventually be treated differently. ie. image if this schema was imported in the dapp and validated there and failed cause this field is not a part of the spec. I think you mentioned it in `validate_field_helper` , but I think it's fine for now.

Reply:
i think it's okay to have fields that are not part of the metadata chain, right? even in the context of the disc prov, we take the entire schema and generate sub schemas to validate fields rather than the entire object. we can apply the same logic to services and only validate what is necessary. that way, we can a la carte what we think is necessary for validation

Reply:
discussed offline -- will remove non meta-data fields as validation is not necessary for them